### PR TITLE
CALS-5574 Facility search - Page header

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to?
-  https://osi-cwds.atlassian.net/browse/CALS-5574

#### Brief feature description
- add breadcrumbs to the body of the page below page header in search.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- N/A

### CODE REVIEW

**Comment by ratneshraval** [comment]
@Nasir2712 remove the spaces from story number

File: test/javascript/components/common/breadCrumb_tests.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import BreadCrumb from 'components/common/breadCrumb'
+import {shallow, mount} from 'enzyme'
+
+describe('bread crumb rendering', () => {
+  let breadCrumbComp
+  it('verify breadCrumb rendering', () => {
+    let props = <a href="/search"> Facility Search</a>
+    breadCrumbComp = mount(<BreadCrumb children={props} />)
```
Comment (ratneshraval) [inline test/javascript/components/common/breadCrumb_tests.js]: Do you need `mount` here vs `shallow`?

File: app/javascript/components/common/breadCrumb.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import {urlPrefixHelper} from 'helpers/url_prefix_helper.js.erb'
+
+const BreadCrumb = ({
+  breadCrumbSign,
+  children
+}) => (
+  <div className='container'>
+    <div className='row'>
+      <div className='col-xs-7'>
+        <h5>Back to: <span><a href={urlPrefixHelper('/')}>Main Menu</a> {breadCrumbSign} {children}</span></h5>
```
Comment (ratneshraval) [inline app/javascript/components/common/breadCrumb.js]: What happens when you're 2 levels deep? Would we need to update this component at that time?

Comment (Nasir2712) [reply app/javascript/components/common/breadCrumb.js]: yes we have too

**Pull request merged**